src/models/user.model.ts

```diff
@@ -0,0 +1,26 @@
+import { DataTypes } from "sequelize";
+
+export enum UserErrors {
+    UserIdDoesNotExist,
+    UserLoginDoesNotExist,
+    UserRemoved,
+};
+
+export const sequelizeUserModel = {
```

Comment:
Do you really need `sequelize` in the variable name?

Reply:
yes, don't see problem to avoid magic number

Reply:
I meant that it should be named `userModel`.